src/main/credentialServer/httpCredentialServer.ts

```diff
@@ -0,0 +1,210 @@
+import fs from 'fs';
```

Comment:
Please add a markdown file in this directory describing the protocol.

Reply:
Good idea -- it's a non-standard protocol.